.github/workflows/ci.yml

```diff
@@ -30,9 +35,17 @@ jobs:
       - run: yarn workspace @foxglove/${{ matrix.package }} lint:ci
       - run: yarn workspace @foxglove/${{ matrix.package }} test
 
+      - run: yarn workspace @foxglove/${{ matrix.package }} pack
+      - name: Publish to NPM (dry run)
+        if: ${{ matrix.node-version == '18.x' }}
+        # `yarn publish` does not support --provenance
```

Comment:
you linked to the yarn berry issue in the mcap repo, would be good to have it here too:
https://github.com/yarnpkg/berry/issues/5430

Reply:
This repo isn’t using berry for better or worse

Reply:
I think you can drop this yarn comment altogether. It isn't crazy that to publish to a repository you use the tool made by that repository.

Reply:
I will keep the comment since I think the `yarn pack && npm publish package.tgz` is a bit unusual otherwise and I would wonder why it's being done this way.